### DISCUSSION TITLE
how-to-monitor-windows-systems-with-netdata/

### DISCUSSION THREAD

**Discussion by giscus** [netdata/blog#244] [General]
# how-to-monitor-windows-systems-with-netdata/

How to monitor Windows systems with Netdata

https://blog.netdata.cloud/how-to-monitor-windows-systems-with-netdata/

**Comment by jhgorse** [comment]
links need refreshed.

**Reply by andrewm4894** [reply]
cheers - think this fixes them: https://github.com/netdata/blog/pull/245

**Comment by shyamvalsan** [comment]
In fact, I think the content needs a refresh as well. Will try and do an update today.
